R/aaa_imports.R

```diff
@@ -0,0 +1,3 @@
+#' Re-imports
+#' @importFrom rlang `%||%`
+`%||%` <- rlang::`%||%`
```

Comment:
Can we not bring `rlang` where `ifelse` works too?
It's a large dependency that adds much overhead

Reply:
`ifelse` won't work with `NULL` or length 0 values. I can just copy the code for `%||%` into an internal function in the package and avoid the dependency.

Reply:
This is done in 00c9e1e